config/galaxy.ini.sample

```diff
@@ -633,6 +638,29 @@ nglims_config_file = tool-data/nglims.yaml
 # https://FQDN/galaxy-prefix/gie_proxy for proxying
 #dynamic_proxy_prefix=gie_proxy
 
+# The Golang proxy also manages the docker containers more closely than the
+# NodeJS proxy, so is able to expose more container management related options
+
+# This attribute governs the minimum length of time between consecutive HTTP/WS
```

Comment:
But surely there is some javascript frontend to the postgres DB that's polling the webpage? :( yeah, it may need to be GIE specific, but perhaps we can have a single global variable that individual IEs can override in their INI file?